config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1060,6 +1062,16 @@ def generate_presubmits_scale():
             scenario='scalability',
             networking='amazonvpc',
             always_run=False,
+        ),
+        presubmit_test(
+            name='presubmit-kops-aws-scale-amazonvpc-using-cl2',
+            scenario='scalability',
+            networking='amazonvpc',
+            always_run=False,
+            artifacts='$(ARTIFACTS)',
+            env={
+                'RUN_CL2_TEST': "true",
+            }
```

Comment:
`artifacts` ends up being an env var in the end, same as `RUN_CL2_TEST`. Why add an extra parameter for it?

Reply:
Initially, I tried adding it as an env variable but the result in the job generated has 
```
- name: ARTIFACTS
   value: "$(ARTIFACTS)"
```
I compared it with other job definitions and to keep it consistent `value: $(ARTIFACTS)` went with this approach of defining a placeholder in the template and hence an extra param.